### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by Hackante**
closes wwosimulation/tracker#170

### CODE REVIEW

**Comment by Hackante** [comment]
It's not ready yet. Can someone convert it to an draft pr? I'm in the app rn.

File: commands/economy/profile.js [hunk 1]
```diff
@@ -1,49 +1,124 @@
...
+        if (guyz.author) guyz = guyz.author
+        let guy = await players.findOne({ user: guyz.id })
+        let inventory = guy.profile
         if (inventory != true && !client.botAdmin(message.author.id)) return message.channel.send(message.l10n("profileNeedToBuy"))
+        if (!guy.profile && !fn.isNarrator(message.member)) return message.channel.send(message.l10n("profileLocked"))
 
-        if (!data.profile && !fn.isNarrator(message.member)) return message.channel.send(message.l10n("profileLocked"))
+        let name = guyz.user ? guyz.user.tag : guyz.tag
+        if (name.length > 20) {
+            name = name.slice(0, 15) + "...#" + message.author.discriminator
```
Comment (thewilloftheshadow) [inline commands/economy/profile.js:56]: Uses the author's discriminator even if the author isn't the target user

**Review by thewilloftheshadow — CHANGES_REQUESTED** [review]

**Comment by Hackante** [comment]
image
fixed everything

**Comment by thewilloftheshadow** [comment]
Might wanna look at a font change in the future but lgtm for now

**Review by thewilloftheshadow — APPROVED** [review]

**Pull request merged**